nncf/common/pruning/operations.py

```diff
@@ -197,14 +199,14 @@ def generate_output_mask(cls, node: NNCFNode, graph: NNCFGraph,
         :return: Filled input masks.
         """
         input_edges = graph.get_input_edges(node)
```

Comment:
```suggestion
```

Reply:
It seems that `input_edges` is used in the line 215.

Reply:
You are right, sorry